Source/MLXNN/Quantized.swift

```diff
@@ -114,11 +114,15 @@ open class QuantizedLinear: Linear {
         model: Module,
         groupSize: Int = 64,
         bits: Int = 4,
-        predicate: (Module) -> Bool = { $0 is Linear }
+        predicate: (Linear) -> Bool = { _ in true }
```

Comment:
This predicate was accidentally not used -- the code already had a filter for being a `Linear`